sw-tests.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This is the file you commented that we can do dynamic on a next iteration, will be easy with gulp, but so far sounds good to keep it static.